e2e/tests-dfx/sns.bash

```diff
@@ -65,22 +65,19 @@ SNS_CONFIG_FILE_NAME="sns.yml"
     dfx start --clean --background --host 127.0.0.1:8080
     sleep 1
```

Comment:
The local entries mentioned and patched on the left are now entered cleanly so we can remove the patchwork.